jekyll/_cci2/jenkins-converter.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: classic-docs
+title: "Introduction to Jenkins Converter"
+short-title: "Jenkins Converter Introduction"
+description: "Starting point for how to use the Jenkins Converter"
+categories: [getting-started]
+order: 1
+---
+## Introduction to Jenkins Converter
+
+The Jenkins Converter is a web tool that allows you to easily convert a Jenkinsfile to a CircleCI config.yml, helping you get started on building on CircleCI quickly and easily. It can be accessed at https://circleci.com/developer/tools/jenkins-converter.
+
+Currently, the converter only supports declarative Jenkinsfiles. While the number of supported plug-ins and steps continue to be expanded, we hope that this tool gets you started at least 50% of the way, and makes it easier for you to get started on building on CircleCI.
+
+## Limitations
+
+* Limited number of syntaxes and plugins are supported. Jenkinsfiles relying on unsupported syntaxes and plugins cannot be converted. Please manually remove them.
+
+* Only `Default` is supported as a tool name for `maven`, `jdk` and `gradle` in the [`tools` block](https://www.jenkins.io/doc/book/pipeline/syntax/#tools), and other names will cause conversion failures. Please configure them appropriately or remove them manually.
+
+  For example, the following stanza:
+  ```
+  tools {
+    maven 'Maven 3.6.3'
+    jdk 'Corretto 8.232'
+  }
+  ```
+  should be changed to:
+  ```
+  tools {
+    maven 'Default'
+    jdk 'Default'
+  }
+  ```
+
+## Next Steps After Conversion
+
+### Executors
+
+For the time being, a static Docker executor, [cimg/base](https://github.com/CircleCI-Public/cimg-base), is inserted as the [executor](https://circleci.com/docs/reference-2-1/#executors) regardless of the one defined within the Jenkinsfile input.
+
+Given that this is a very lean image, it's highly likely that your project will require a different executor. [CircleCI's convenience images](https://circleci.com/docs/2.0/circleci-images/) are a good place to get started, while more advanced users may be ready to base their project off a [custom Docker image](https://circleci.com/docs/2.0/custom-images/).
+
+Depending on the use case, you may require the usage of a [machine executor](https://circleci.com/docs/2.0/executor-types/#using-machine) when your application requires full access to OS resources and the job environment, or the [macOS executor](https://circleci.com/docs/2.0/executor-types/#using-macos).
+
+### Workflows
+
+[CircleCI Workflows](https://circleci.com/docs/2.0/workflows/) (the equivalent of Jenkins pipelines) should be fully transferred from your Jenkinsfile to config.yml, including branch filters. We've made a conscious decision to not transfer over any [scheduled builds](https://circleci.com/docs/2.0/configuration-reference/#triggers) to prevent unintentional builds from being triggered.
```

Comment:
```suggestion
[CircleCI Workflows](https://circleci.com/docs/2.0/workflows/) (the equivalent of Jenkins pipelines) should be fully transferred from your Jenkinsfile to config.yml, including branch filters. The converter will not transfer over any [scheduled builds](https://circleci.com/docs/2.0/configuration-reference/#triggers) to prevent unintentional builds from being triggered.
```